packages/ui-components/src/components/time-picker/time-picker.tsx

```diff
@@ -83,12 +83,19 @@ export class KvTimePicker implements ITimePicker, ITimePickerEvents {
 	@Event() dropdownStateChange: EventEmitter<boolean>;
 	/** @inheritdoc */
 	@Event() cancelClicked: EventEmitter<CustomEvent<MouseEvent>>;
+	/** @inheritdoc */
+	@Event() showCalendarStateChange: EventEmitter<boolean>;
 
 	@Watch('selectedTimeOption')
 	handleSelectTimeStateChange(timeState: ITimePickerTime) {
 		this.selectedTimeState = timeState;
 	}
 
+	@Watch('showCalendar')
+	handleShowCalendarChange() {
```

Comment:
This will not be called when the component is mounted, is this the expected behaviour?

Reply:
I call the syncShowCalendarViewState in the componentWillLoad lifecycle method as well, thus, covering the mount case. 
However, while testing by setting the prop showCalendar in the storybook interface, I noticed that the state of the prop did not affect the view of the component because we were not watching changes to this prop. Therefore I added this Watch to cover that and possible future cases, although, I believe It will not happen but its done just in case 👍